registry/recipes/packages/py-prompthandler/conftest.py

```
@@ -0,0 +1,93 @@
+
+    def __init__(self, prompts: dict[str, str] | None = None) -> None:
+        """
+        Initialize a Prompt handler that saves prompts in memory.
+
+        Args:
+            prompts (dict[str, str] | None, optional): A dictionary of prompts, If None an empty dictionary is used meaning there are no existing prompts yet. Defaults to None.
+        """
+        super().__init__()
+        self.prompts = prompts or dict()
+
+    def create_prompt(self, name: str, content: str, *args, **kwargs):
```

Ruff (0.14.14)

[warning] Unused method argument: `args`

(ARG002)